frontend-utils/src/player_options/read.rs

```diff
@@ -57,6 +57,12 @@ pub fn read_player_options<'a>(
     // Spoof Url
     result.spoof_url = table.parse_from_str(cx, "spoof_url");
 
+    // Referer
+    result.referer = table.parse_from_str(cx, "referer");
+
+    // Cookie
+    result.cookie = table.parse_from_str(cx, "cookie");
+
```

Comment:
I am not sure if we want these fields in bundle format just yet..

WDYT @Dinnerbone?

Reply:
Hmmmm, okay. @wsxarcher maybe for now remove the deserialization and we can follow up on that later

Reply:
done!